### PR TITLE
Rework themesEqual() to use fuzzy equality on alpha channels  

### PR DESCRIPTION
**Pull request by lmorchard**
JS FP WTF FML

### CODE REVIEW

**Comment by lmorchard** [comment]
Or in other words, I broke down and rewrote `themesEqual()` to be more explicit than a magical JSON string compare and added a fuzz factor to alpha channels because apparently `0.86` is sometimes `0.8599999999901` or some nonsense

**Comment by johngruen** [comment]
Okay looking at this now

__takes belt of whiskey__

File: src/lib/themes.js [hunk 1]
```diff
@@ -1,13 +1,79 @@
...
-  // HACK: "deep equal" via stringify
-  // http://www.mattzeunert.com/2016/01/28/javascript-deep-equal.html
-  JSON.stringify(themeA) === JSON.stringify(themeB);
+export const themesEqual = (themeA, themeB) => {
+  if (!!themeA !== !!themeB) {
+    return false;
+  }
+
+  const hasImagesA =
+    "images" in themeA && "additional_backgrounds" in themeA.images;
```
Comment (johngruen) [inline src/lib/themes.js]: TIL you can do this

**Comment by johngruen** [comment]
works/makes sense!

**Comment by johngruen** [comment]
@lmorchard shi

**Pull request merged**